ikarus/finiteelements/mechanics/materials/vanishingstress.hh

```diff
@@ -199,11 +199,7 @@ private:
 
     int minIter = isAutoDiff ? 1 : 0;
     // THE CTAD is broken for designated initializers in clang 16, when we drop support this can be simplified
-    NewtonRaphsonConfig<decltype(linearSolver), decltype(updateFunction)> nrs{
-        .parameters     = {.tol = tol_, .maxIter = 100, .minIter = minIter},
-        .linearSolver   = linearSolver,
-        .updateFunction = updateFunction
-    };
+    NewtonRaphsonConfig nrs({.tol = tol_, .maxIter = 100, .minIter = minIter}, linearSolver, updateFunction);
```

Comment:
Ok can you move the CTAD comment from here to the deduction guides inside the newton raphson files to explain why these deduction guides are needed.